types/tsqlstring/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for tsqlstring 1.0
+// Project: https://github.com/kylefarris/tsqlstring
+// Definitions by: Dong Joon Seo <https://github.com/srencat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
```

Comment:
```suggestion
```
It's already TS4.0 used here, let's just skip this line

Reply:
thx!